app/Models/User.php

```diff
@@ -109,30 +138,30 @@
  * @property int $user_avatar_type
  * @property int $user_avatar_width
  * @property string $user_birthday
- * @property string $user_colour
+ * @property string|null $user_colour
  * @property string $user_dateformat
- * @property mixed $user_discord
+ * @property string|null $user_discord
  * @property int $user_dst
  * @property string|null $user_email
- * @property mixed $user_email_confirmation
+ * @property-write string|null $user_email_confirmation
  * @property int $user_emailtime
- * @property string $user_from
+ * @property string|null $user_from
  * @property int $user_full_folder
  * @property int $user_id
  * @property int $user_inactive_reason
  * @property int $user_inactive_time
  * @property string|null $user_interests
  * @property string $user_ip
- * @property string $user_jabber
+ * @property string|null $user_jabber
  * @property string $user_lang
  * @property string $user_last_confirm_key
  * @property int $user_last_privmsg
  * @property int $user_last_search
  * @property int $user_last_warning
- * @property int $user_lastmark
+ * @property Carbon|null $user_lastmark
  * @property string $user_lastpage
- * @property int $user_lastpost_time
- * @property int $user_lastvisit
+ * @property Carbon|null $user_lastpost_time
+ * @property Carbon|null $user_lastvisit
```

Comment:
`user_lastfm` and `user_lastfm_session` are missing...

Although I wonder if those unused/unwanted attributes should be made inaccessible instead (and commented out in `getAttribute` instead of deleted so it's not accidentally added back unless actually needed)

Reply:
I put them in for now but I think we should consider removing unused attributes since it just adds clutter if they're unused (and also avoid accidentally using them 👀 )